ddtrace/contrib/grpc/client_interceptor.py

```diff
@@ -102,7 +103,7 @@ def _handle_error(span, response_error, status_code):
             exc_val = to_unicode(response_error.details())
             span._set_str_tag(errors.ERROR_MSG, exc_val)
             span._set_str_tag(errors.ERROR_TYPE, status_code)
```

Comment:
Should we `stringify` here too? Could the status code be an int?

Reply:
(Also we should check all the usages of _set_str_tag to be sure that there aren't any other possibilities like this

Reply:
`status_code` has already been coerced into unicode in this integration. Other integrations using `_set_str_tag` were already updated in this PR.